src/index.html

```diff
@@ -7,12 +7,74 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <section
+      data-qa="card"
+      class="card"
+    >
+      <div class="card__product">
+        <div class="card__product-img card__product-img--position">
+          <img
+            src="./images/imac.jpeg"
+            alt="APPLE A1419 iMac 27 Retina 5K (MNED2UA/A)"
+            class="card__img"
+          />
+        </div>
+
+        <div class="card__product-name">
+          <h2 class="card__name">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+        </div>
+
+        <p class="card__code">Product code: 195434</p>
+
+        <div class="card__product-stars">
+          <div class="card__stars">
+            <section class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </section>
+
+            <p class="card__stars-reviews">Reviews: 5</p>
+          </div>
+        </div>
+
+        <div class="card__product-price">
+          <div class="card__price">
+            <p class="card__price-txt">Price:</p>
+            <p class="card__price-sum">$2,199</p>
+          </div>
+        </div>
+
+        <a
+          href="#"
+          class="card__button-buy"
+          data-qa="hover"
+        >
+          Buy
+        </a>
+      </div>
```

Comment:
The div elements are used more than necessary. Div is a non-semantic element, and it's better to use semantic elements where possible. It helps to make the structure of the page more understandable and accessible. For example, instead of using div for 'card__product-name' and 'card__product-stars', you can use more appropriate semantic elements.